content/zh/includes/default-storage-class-prereqs.md

```diff
@@ -1,4 +1,4 @@
-您需要有一个带有默认[StorageClass](/docs/concepts/storage/storage-classes/)的动态持续卷供应程序，或者自己[静态的提供持久卷](/docs/user-guide/persistent-volumes/#provisioning)来满足这里使用的[持久卷请求](/docs/user-guide/persistent-volumes/#persistentvolumeclaims)。
+您需要有一个带有默认[StorageClass](/content/zh/docs/concepts/storage/storage-classes.md)的动态持续卷供应程序，或者自己[静态的提供持久卷](/content/zh/docs/reference/glossary/persistent-volume.md)来满足这里使用的[持久卷请求](/content/zh/docs/reference/glossary/persistent-volume-claim.md)。
```

Comment:
1. Most zh translation docs still keep the links to en version. A general fix for this is under discussion.
1. No need for `.md` in a link.